Cargo.toml

```diff
@@ -42,8 +42,11 @@ qt-build-utils = { path = "crates/qt-build-utils", version = "0.5.3" }
 
 cc = { version = "1.0.79", features = ["parallel"] }
 # Ensure that the example comments are kept in sync
-# examples/cargo_without_cmake/Cargo.toml
-# examples/qml_minimal/rust/Cargo.toml
+# ./examples/cargo_without_cmake/Cargo.toml
+# ./examples/qml_minimal/rust/Cargo.toml
+# ./book/src/getting-started/4-cargo-executable.md
+# ./book/src/getting-started/5-cmake-integration.md
+# TODO: Can we re-export cxx from cxx-qt, so people don't need to manually add this anymore?
```

Comment:
probably yes, question will be does a `#[cxx_qt::cxx::bridge]` macro work in our expansion, or would we put a `use cxx_qt::cxx;` or something? :thinking: something to play about with in the next cycle, add an issue so we don't forget